main/EDMainMod.F90

```diff
@@ -835,6 +835,9 @@ subroutine ed_update_site( currentSite, bc_in, bc_out, is_restarting )
     type (fates_patch_type) , pointer :: currentPatch
     !-----------------------------------------------------------------------
 
+    ! check patch order (set second argument to true)
+    ! call set_patchno(currentSite,.true.,1)
+    
```

Comment:
Did you want to keep this in just as a check?  I'm assuming it technically can be removed since you're calling `set_patchno` at the end of `ed_ecosystem_dynamics` via `terminate_patches` (which is a good move IMO).

Reply:
yes, or maybe we could put it behind a local debug flag. It is not needed, it is more of a developer level check